roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -58,3 +58,7 @@ dashboard_master_toleration: true
 # Override dashboard default settings
 dashboard_token_ttl: 900
 dashboard_skip_login: false
+
```

Comment:
Maybe put those variables after `coredns`and `nodelocaldns` to keep dns-oriented variables on top and dashboard ones at the bottom